cpu-cycles/Cargo.toml

```diff
@@ -4,9 +4,10 @@ version = "0.1.0"
 edition = "2021"
 build = "build.rs"
 links = "cpucycles"
+license = "LicenseRef-PD-hp OR CC0-1.0 OR 0BSD OR MIT-0 OR MIT"
```

Comment:
@durch just confirming this seems correct for cpu-cycles? It's the same license as I found for libcpucycles

Reply:
@durch prompt :)

Reply:
👍